src/SprykerSdk/Zed/AppSdk/Communication/Console/AppTranslationBuilderConsole.php

```diff
@@ -0,0 +1,142 @@
+<?php
+
+/**
+ * Copyright © 2019-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerSdk\Zed\AppSdk\Communication\Console;
+
+use Generated\Shared\Transfer\AppTranslationRequestTransfer;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Question\ChoiceQuestion;
+use Symfony\Component\Console\Question\Question;
+
+/**
+ * @method \SprykerSdk\Zed\AppSdk\Business\AppSdkFacadeInterface getFacade()
+ */
+class AppTranslationBuilderConsole extends AbstractConsole
+{
+    /**
+     * @var array
+     */
+    protected $translations = [];
+
+    /**
+     * @var string
+     */
+    public const TRANSLATION_FILE = 'translation-file';
+
+    /**
+     * @var string
+     */
+    public const TRANSLATION_FILE_SHORT = 't';
+
+    /**
+     * @return void
+     */
+    protected function configure(): void
+    {
+        $this->setName('app:translation:create')
+        ->setDescription('Create translation file.')
+        ->addOption(static::TRANSLATION_FILE, static::TRANSLATION_FILE_SHORT, InputOption::VALUE_REQUIRED, '', $this->getConfig()->getDefaultAppTranslationFile());
+    }
+
+    /**
+     * @param \Symfony\Component\Console\Input\InputInterface $input
+     * @param \Symfony\Component\Console\Output\OutputInterface $output
+     *
+     * @return int
+     */
+    public function execute(InputInterface $input, OutputInterface $output): int
+    {
+        $appTranslationRequestTransfer = new AppTranslationRequestTransfer();
+
+        $appTranslationRequestTransfer->setConfigurationFile($input->getOption(static::TRANSLATION_FILE));
```

Comment:
```suggestion
        $appTranslationRequestTransfer->setTranslationFile($input->getOption(static::TRANSLATION_FILE));
```
Check the other places as well.